node/lib/package/LocalPackageSource.ts

```diff
@@ -319,14 +320,112 @@ export class LocalPackageSource extends PackageSource {
                 onboarding: onboardingModulePath,
                 onboardingProperties: onboardingProperties,
             }],
+            onboardingInfo: [],
             version: packageJson.version,
         };
     }
 
     private async loadOnboardingPackageInfoAsync(
-            onboardingName: string): Promise<PackageInfo | null> {
-        // TODO: Load onboarding package info from XML + package.json
-        return Promise.resolve(null);
+        onboardingName: string): Promise<PackageInfo | null> {
+
+        // By convention there are manifest.xml and package.json files under 'node'' directory. 
+        // under a directory with the translator name, under the schema directory.
+        // And there should be manifest.xml and package.json files alongside it.
+        let onboardingManifestPath = onboardingName + "/node/manifest.xml";
+        let onboardinPackageJsonPath = onboardingName + "/node/package.json";
+
+        if (!(await fs.exists(path.join(this.sourceDirectory, onboardingManifestPath))) ||
+            !(await fs.exists(path.join(this.sourceDirectory, onboardinPackageJsonPath)))) {
+            // The requested translator package was not found.
+            return null;
+        }
+
+        let packageJson: any = await this.loadJsonAsync(onboardinPackageJsonPath);
+        let manifestXml: any = await this.loadXmlAsync(onboardingManifestPath);
+        let manifestXmlRoot: any = manifestXml && manifestXml.manifest;
+
+        if (!packageJson || !manifestXmlRoot) {
+            return null;
+        }
+
+        // Parse schema info from the manifest.
+        let schemaInfos: any[] = [];
+        let schemaModulePaths: string[] = [];
+        if (Array.isArray(manifestXmlRoot.schemas) &&
+                manifestXmlRoot.schemas.length == 1 &&
+                Array.isArray(manifestXmlRoot.schemas[0].schema)) {
+            let schemaElements = manifestXmlRoot.schemas[0].schema;
+           // TODO: Will onboarding schemas also have 'main' schema concept ?
+
+            schemaElements.forEach((schemaElement: any) => {
+                let schemaId: string = schemaElement.$.id;
+                if (schemaId) {
+                    // By convention the schema module file and its parent directory
+                    // both match the name of the schema.
```

Comment:
Update the comment: the assumption is onboarding schemas are always in the same directory as the onboarding module?